homeassistant/components/fibaro/climate.py

```diff
@@ -125,7 +125,7 @@ def __init__(self, fibaro_device):
         self._op_mode_device = None
         self._fan_mode_device = None
         self._support_flags = 0
-        self.entity_id = f"climate.{self.ha_id}"
+        self.entity_id = ENTITY_ID_FORMAT.format(self.ha_id)
```

Comment:
This is already here so we probably shouldn't change it in this PR, but usually we set the name and let Home Assistant pick the entity id.